pkg/controllers/queue/queue_controller_action.go

```diff
@@ -42,10 +43,17 @@ func (c *queuecontroller) syncQueue(queue *schedulingv1beta1.Queue, updateStateF
 		// Ignore error here, tt can not occur.
 		ns, name, _ := cache.SplitMetaNamespaceKey(pgKey)
 
-		// TODO: check NotFound error and sync local cache.
 		pg, err := c.pgLister.PodGroups(ns).Get(name)
 		if err != nil {
-			return err
+			if !errors.IsNotFound(err) {
+				return err
+			}
+
+			klog.V(4).Infof("sync local cache since podGroup %s not found.", pgKey)
```

Comment:
>>> klog.V(4).Infof("sync local cache since podGroup %s not found.", pgKey)
klog.V(4).Infof("The podGroup %s is not found, skip it and continue to syne cache", pgKey)